src/platform/packages/shared/response-ops/rule_params/anomaly_detection/index.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+export { mlAnomalyDetectionAlertParamsSchema } from './latest';
+export { mlAnomalyDetectionAlertParamsSchema as mlAnomalyDetectionAlertParamsSchemaV1 } from './v1';
+
+export { jobsSelectionSchema } from './latest';
```

Comment:
this is probably a leftover as `jobsSelectionSchema ` is in `common/utils`

Reply:
yes, the TS issue needs to be fixed
```
Module '"./latest"' has no exported member 'jobsSelectionSchema'.
```

Reply:
Fixed!